package-lock.json

```diff
@@ -46,11 +46,12 @@
             }
         },
         "../websoc-fuzzy-search": {
-            "version": "0.6.3",
+            "version": "0.7.0",
             "extraneous": true,
             "license": "MIT",
             "devDependencies": {
-                "prettier": "^2.6.2"
+                "prettier": "^2.6.2",
+                "typescript": "^4.6.3"
```

Comment:
What's with typescript on the dev dependencies?

Reply:
This shouldn't have happened—not sure how that got in there, probably WebStorm trying to be "helpful." I'll go ahead and revert that.

Reply:
Wait no that's a fuzzy search dependency.

The latest version on the repo does have TypeScript as a devDependency because it has TS type declarations, but it looks like switching back to v0.6.3 didn't get rid of the entry for the local package in the `package-lock.json`. I'll see if I can remove it without breaking everything.